Live: adding docs (#34642)
grafana/grafana: docs/sources/live/_index.md

+++

> title = "Live"

Reviewer: ```suggestion
title = "Grafana Live"
```

aliases = []
weight = 115
+++

# Grafana Live overview

> Grafana Live is a real-time messaging engine introduced in Grafana v8.

Reviewer 2: ```suggestion
Grafana Live is a real-time messaging engine introduced in Grafana v8.
```
```suggestion
Grafana Live is a real-time messaging engine introduced in Grafana v8.0.
```

> With Grafana Live it's possible to push event data to a frontend as soon as some event happened.

Reviewer 2: ```suggestion
With Grafana Live, you can push event data to a frontend as soon as an event occurs.
```

> This could be notifications about dashboard changes, new frames for rendered data etc. Live features can help to eliminate a page reload or polling in many places, can be used to stream IOT sensor or any other real-time data to panels.

Reviewer: ```suggestion
This could be notifications about dashboard changes, new frames for rendered data, etc. Live features can help eliminate a page reload or polling in many places, it can be used to stream Internet of things (IOT) sensors or any other real-time data to panels.
```

> This could be notifications about dashboard changes, new frames for rendered data, etc. Live features can help eliminate a page reload or polling in many places, it can be used to stream Internet of things (IOT) sensors or any other real-time data to panels.

Reviewer 2: ```suggestion
This could be notifications about dashboard changes, new frames for rendered data, and so on. Live features can help eliminate a page reload or polling in many places, it can stream Internet of things (IOT) sensors or any other real-time data to panels.
```

> > When saying `real-time` in this doc we mean a soft real-time. Due to network latencies, garbage collection cycles etc. the delay of delivered message can be up to several hundred milliseconds or higher on practice.

Reviewer: ```suggestion
> **Note:** By `real-time`, we indicate a soft real-time. Due to network latencies, garbage collection cycles, etc., the delay of a delivered message can be up to several hundred milliseconds or higher on practice.
```

> > **Note:** By `real-time`, we indicate a soft real-time. Due to network latencies, garbage collection cycles, etc., the delay of a delivered message can be up to several hundred milliseconds or higher on practice.

Reviewer 2: ```suggestion
> **Note:** By `real-time`, we indicate a soft real-time. Due to network latencies, garbage collection cycles, and so on, the delay of a delivered message can be up to several hundred milliseconds or higher.
```
